tpm2tools/misc.go

```diff
@@ -0,0 +1,32 @@
+package tpm2tools
+
+import (
+	"io"
+
+	"github.com/google/go-tpm-tools/proto"
+	"github.com/google/go-tpm/tpm2"
+)
+
+// ReadPCRs fetchs the values of the specified PCRs for the specified hash.
+func ReadPCRs(rw io.ReadWriter, pcrs []int, hash tpm2.Algorithm) (*proto.Pcrs, error) {
```

Comment:
Does this need to return a proto? Right now we only use protos when we have to pass data to/from the disk or to/from the server.

Right now we normally just pass PCR values as a map from index to value. Would that work as the return value here?

Reply:
Obviously that would work, but the pattern established elsewhere is to return protos where possible. I find this somewhat natural for testability.

For example, seal returns the sealedbytes proto instead of an equivalent go structure. Note, this contains more than just the map, it's also tagged with the hash algorithm. Not that it is hard to return both pieces of data, it's that this seems pretty comparable.